app/templates/checkout-web.html

```diff
@@ -0,0 +1,247 @@
+    <div class="flex flex-row min-h-[50%]">
+        <div class="basis-2/4">
+            <h2 class="text-3xl">Instructions</h2>
+            <div class="mt-3 pr-10">
+                <p>
+                    Checkout form will send requests directly to VGS Proxy to create financial instrument and perform money transfer.
+                </p>
+                <p class="mt-3">
+                    Fill out the form, click on "Save Card" checkbox and click on "ADD PAYMENT METHOD".
+                    The Universal Checkout will tokenize the sensitive card data.
+                </p>
+
+                <div class="stages mt-10">
+                    <div class="stage waiting" stage="create-financial-instrument">
+                        Create a financial instrument
+                    </div>
+                    <div class="stage waiting" stage="check-saved-cards">
+                        Check saved cards
+                    </div>
+                    <div class="stage waiting" stage="transfer-money">
+                        Perform money transfer
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="basis-2/4">
+            <div class="w-[450px] shadow-md rounded-md">
+                <div class="demo-data p-10 bg-slate-200">
+                    <h2 class="text-xl ">Example Credit Card Data</h2>
+                    <div class="flex flex-row">
+                        <span class="opacity-50">Cardholder Name:&nbsp;</span>
+                        <span class="">Thomas Anderson</span>
+                    </div>
+                    <div class="flex flex-row">
+                        <span class="opacity-50">Card number:&nbsp;</span>
+                        <span class="">4242 4242 4242 4242</span>
+                    </div>
+                    <div class="flex flex-row">
+                        <span class="opacity-50">Expiration Date:&nbsp;</span>
+                        <span class="">06/25</span>
+                    </div>
+                    <div class="flex flex-row ">
+                        <span class="opacity-50">CVC:&nbsp;</span>
+                        <span class="">100</span>
+                    </div>
+                </div>
+                <hr>
+                <div class="m-5">
+                    <a href="" id="reload">Reset Form</a>
+                </div>
+                <div id="vgs-checkout"></div>
+            </div>
+        </div>
+    </div>
+    <div id="server-logs-container" class="mt-20">
+        <h2 class="text-xl font-bold ">Logs</h2>
+        <div id="server-logs">
+            <div class="server-log-event">Loaded index.html page.</div>
+            <div class="server-log-event">Waiting for form submission...</div>
+        </div>
+    </div>
+</div>
+
+<script src="https://checkout.verygoodvault.com/v1/"></script>
+<script type="text/javascript">
+    initCheckoutForm();
+
+    async function initCheckoutForm() {
```

Comment:
This is awesome. Great work following the pattern from previous page. Can we also add in an example and log entry of how to post a transaction with this model?